services/galley/src/Galley/Cassandra/Team.hs

```diff
@@ -77,7 +77,6 @@ interpretTeamStoreToCassandra lh = interpret $ \case
   CreateTeam t uid n i k b -> embedClient $ createTeam t uid n i k b
   DeleteTeamMember tid uid -> embedClient $ removeTeamMember tid uid
```

Comment:
I would leave this change (the whole file) in, not revert it.

Reply:
Fixed.